tests/sentry/notifications/utils/test_participants.py

```diff
@@ -331,10 +361,177 @@ def test_fallthrough(self):
+            data={
+                "stacktrace": {
+                    "frames": [
+                        {
+                            "function": "handledError",
+                            "abs_path": "Application.lol",
+                            "module": "io.sentry.example.Application",
+                            "in_app": True,
+                            "lineno": 39,
+                            "filename": "Application.lol",
+                        },
+                    ]
+                },
+            },
+            project_id=project_no_team.id,
+        )
+
+        GroupOwner.objects.create(
+            group=event.group,
+            user=self.user3,
+            project=self.project,
+            organization=self.organization,
+            type=GroupOwnerType.SUSPECT_COMMIT.value,
+            context={"commitId": self.commit.id},
+        )
+        assert self.get_send_to_owners(event) == {
+            ExternalProviders.EMAIL: {
+                user_service.get_user(self.user3.id),
+            },
+            ExternalProviders.SLACK: {
+                user_service.get_user(self.user3.id),
+            },
+        }
+
+    @with_feature("organizations:streamline-targeting-context")
+    @with_feature("organizations:commit-context")
+    def test_send_to_suspect_committers_dupe_with_commit_context_feature_flag(self):
+        self.commit = self.create_sample_commit(self.user)
+        event = self.store_event(
+            data={
+                "stacktrace": STACKTRACE,
+            },
+            project_id=self.project.id,
+        )
+
+        GroupOwner.objects.create(
+            group=event.group,
+            user=self.user,
+            project=self.project,
+            organization=self.organization,
+            type=GroupOwnerType.SUSPECT_COMMIT.value,
+            context={"commitId": self.commit.id},
+        )
+        assert self.get_send_to_owners(event) == {
+            ExternalProviders.EMAIL: {user_service.get_user(self.user.id)},
+            ExternalProviders.SLACK: {user_service.get_user(self.user.id)},
+        }
+
+    @with_feature("organizations:streamline-targeting-context")
+    @with_feature("organizations:commit-context")
+    def test_send_to_suspect_committers_exception_with_commit_context_feature_flag(self):
+        invalid_commit_id = 10000
+        event = self.store_event(
+            data={
+                "stacktrace": STACKTRACE,
+            },
+            project_id=self.project.id,
+        )
+
+        GroupOwner.objects.create(
+            group=event.group,
+            user=self.user3,
+            project=self.project,
+            organization=self.organization,
+            type=GroupOwnerType.SUSPECT_COMMIT.value,
+            context={"commitId": invalid_commit_id},
+        )
+        assert self.get_send_to_owners(event) == {
+            ExternalProviders.EMAIL: {
+                user_service.get_user(self.user.id),
```

Comment:
`user` owns the project and is the suspect committer, we only want the user to appear once